plugins/generic/externalFeed/templates/homepage.tpl

```diff
@@ -0,0 +1,50 @@
+{**
+* plugins/generic/externalFeed/templates/homepage.tpl
+*
+* Copyright (c) 2003-2013 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* Template for external feed content on journal homepage
+*}
+
+<div id="externalFeedsHome">
+
+	{foreach from=$entries item=entry}
+		<h3>{$entry.feedTitle}</h3>
+		
+		<table class="externalFeeds">
+			<tr>
+				<td colspan="2" class="headseparator">&nbsp;</td>
+			</tr>
+			
+			{assign var="separator" value=''}
+			
+			{foreach from=$entry.items item=item}
+				{$separator}

```

Comment:
You could get rid of `$separator` and instead use the Smarty `first` property:
http://www.smarty.net/docsv2/en/language.function.foreach.tpl#foreach.property.first